code/modules/research/xenoarchaeology/finds/finds.dm

```diff
@@ -96,7 +96,7 @@
 	if(new_item_type)
 		find_type = new_item_type
 	else
-		find_type = rand(1,34)	//update this when you add new find types
+		find_type = rand(1,38)	//update this when you add new find types
```

Comment:
Add a define for this in 
code/modules/research/xenoarchaeology/finds/finds_defines.dm